app/src/main/java/org/fossasia/openevent/general/auth/SignUpFragment.kt

```diff
@@ -140,4 +161,17 @@ class SignUpFragment : Fragment() {
             else -> super.onOptionsItemSelected(item)
         }
     }
+
+    private fun setSignUpState() {
```

Comment:
This should also check if the password is same then only allow the user to click

Reply:
Done @nikit19